File: basetoolslibrary/src/main/java/com/excellence/basetoolslibrary/databinding/OnItemFocusChangeListener.java
package com.excellence.basetoolslibrary.databinding;

import android.view.View;

import androidx.databinding.ViewDataBinding;

/**
 * <pre>
 *     author : VeiZhang
 *     blog   : http://tiimor.cn
 *     time   : 2020/8/24
 *     desc   :
 * </pre> 
 */
public interface OnItemFocusChangeListener {
    void onItemFocusChange(ViewDataBinding binding, View v, boolean hasFocus, int position);
}

File: basetoolslibrary/src/main/java/com/excellence/basetoolslibrary/databinding/OnItemFocusChangeListener.kt
package com.excellence.basetoolslibrary.databinding

import android.view.View
import androidx.databinding.ViewDataBinding

/**
 * <pre>
 *     author : VeiZhang
 *     blog   : http://tiimor.cn
 *     time   : 2020/8/24
 *     desc   :
 * </pre>
 */
interface OnItemFocusChangeListener {

    fun onItemFocusChange(binding: ViewDataBinding, v: View, hasFocus: Boolean, position: Int)

}